drivers/hmis/lib/form_data/fragments/non_cash_benefits.json

```diff
@@ -23,32 +23,32 @@
             {
               "question": "4.03.3",
               "operator": "EQUAL",
-              "answer_boolean": true
+              "answer_code": "YES"
```

Comment:
These changes –– changing a bunch of field types from booleans to NoYesMissing –– is super not backwards compatible. I'll have to destroy existing forms when I deploy this, or else re-opening saved forms will totally crash. 

At this stage in the game I think it's still OK (and necessary, in this case), but at some point we should start incrementing definition versions instead of replacing them. just so you know @eanders